unittests/test_utility.py

```diff
@@ -225,6 +234,12 @@ def test_virtual_copy_linkself(self):
         self.assertRaises(OSError, os_ext.copytree_virtual,
                           self.prefix, self.target, file_links)
 
+    def test_virtual_copy_src_parent_of_dst(self):
```

Comment:
I don't think this test is needed, since they way you call it is equivalent to the `copytree`.